src/main/java/com/atlauncher/Launcher.java

```diff
@@ -456,7 +456,6 @@ public Window getParent() {
      * Reloads the panel used for Instances
      */
     public void reloadInstancesPanel() {
-        InstanceManager.post();
```

Comment:
Can we remove this method entirely? As well as the `setInstancesPanel` since I believe that's the only reason they were used was for when instances we're added/removed

Reply:
Sure, I'll do a quick cleanup when I get home tonight ^-^)